lib/components/dashboard/Navigation/TeamNav.tsx

```diff
@@ -13,14 +13,14 @@ const routes = [
 	},
 	{
 		name: 'Settings',
-		path: 'settings/general'
```

Comment:
This fixes the tab not being highlighted on the Settings page. Still routes properly.